packages/shared/src/authz/policies/project/canCreateModel.spec.ts

```diff
@@ -0,0 +1,359 @@
+import cryptoRandomString from 'crypto-random-string'
+import { assert, describe, expect, it } from 'vitest'
+import { canCreateModelPolicy } from './canCreateModel.js'
+import { parseFeatureFlags } from '../../../environment/index.js'
+import { Roles } from '../../../core/constants.js'
+import { Workspace } from '../../domain/workspaces/types.js'
+import { WorkspacePlan } from '../../../workspaces/index.js'
+import { Authz } from '../../../index.js'
```

Comment:
why? you can get all of the types directly from their modules, no need to have this namespacing internally within the poackage